todo.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script type="text/javascript">
+</script>
+</head>
+
+<body>
+<h1>Todo List</h1>
+    <div id="todoList"></div>
+<hr>
```

Comment:
Here is OK since we don't have CSS, but typically you don't want to use HRs for spacing. So when you introduce CSS you should remove this.